dogu.json

```diff
@@ -46,5 +46,18 @@
   "ExposedCommands": [{
     "Name": "post-upgrade",
     "Command": "/post-upgrade.sh"
-  }]
+  },{
+    "Name": "upgrade-notification",
+    "Command": "/upgrade-notification.sh"
+  },{
+    "Name": "pre-upgrade",
+    "Command": "/pre-upgrade.sh"
+  }],
+  "Configuration": [
```

Comment:
This one is so important that we need it a second time? :D

Reply:
Nope :D